RecoTauTag/RecoTau/plugins/PFRecoTauLostTrackPlugin.cc

```diff
@@ -0,0 +1,83 @@
+/*
+ * =============================================================================
+ *       Filename:  PFRecoTauLostTrackPlugin.cc
+ *
+ *    Description: Add references to tracks of tau-charged-hadrons built on 
+ *                 top of a track
+ *
+ *        Created:  25/04/2022
+ *
+ *         Authors:  Michal Bluj (NCBJ, Warsaw)
+ *
+ * =============================================================================
+ */
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "RecoTauTag/RecoTau/interface/RecoTauBuilderPlugins.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadron.h"
+#include "DataFormats/TauReco/interface/PFRecoTauChargedHadronFwd.h"
+
+namespace reco {
+  namespace tau {
+
+    class PFRecoTauLostTrackPlugin : public RecoTauModifierPlugin {
+    public:
+      explicit PFRecoTauLostTrackPlugin(const edm::ParameterSet&, edm::ConsumesCollector&& iC);
+      ~PFRecoTauLostTrackPlugin() override;
+      void operator()(PFTau&) const override;
+      void beginEvent() override;
+      void endEvent() override;
+
+    private:
+      edm::Handle<reco::TrackCollection> tracks_;
+      edm::EDGetTokenT<reco::TrackCollection> track_token_;
+      int verbosity_;
+    };
+
+    PFRecoTauLostTrackPlugin::PFRecoTauLostTrackPlugin(const edm::ParameterSet& cfg, edm::ConsumesCollector&& iC)
+        : RecoTauModifierPlugin(cfg, std::move(iC)) {
+      track_token_ = iC.consumes(cfg.getParameter<edm::InputTag>("trackSrc"));
+      verbosity_ = cfg.getParameter<int>("verbosity");
```

Comment:
This should be "untracked"

Reply:
In principle yes, but in plugin calling this and similar ones it is the "verbose" is defined as tracked. In a reault this small change in this plugin will propagate to other plugins which I think is not worth an effort. Therefore, I will keep it as it is.